.github/workflows/test.yml

```diff
@@ -14,13 +14,14 @@ jobs:
     - uses: actions/checkout@v2
     - uses: cachix/install-nix-action@v12
       with:
-        skip_adding_nixpkgs_channel: true
+        nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
This is naughty.  I meant to pin this, but wanted to see it working before the hassle of sharing the pin as a variable across the steps.